config/locales/devise_invitable.sv.yml

```diff
@@ -0,0 +1,25 @@
+sv:
+  devise:
+    failure:
+      invited: "Du har en väntande inbjudan, acceptera den för att slutföra skapa ditt konto."
+    invitations:
+      send_instructions: "Ett inbjudningsbrev har skickats till %{email}."
+      invitation_token_invalid: "Inbjudan tillkännagivandet är inte giltigt!"
```

Comment:
`Inbjudan tillkännagivandet är inte giltigt!` => `Nyckeln i inbjudan är inte giltig!`

(Here I called the "token" a "key", perhaps that's acceptable?)

Reply:
As a point of interest, Devise just uses "token" such as:
```
reset_password_token: "Återställ token för lösenord"
```
And
```
invalid_token: 'Ogiltig token för autentisering.'
```